JobBot-server/jobbot/jobbot/src/main/java/org/dizzybot/jobbot/entities/Job.java

```diff
@@ -0,0 +1,71 @@
+package org.dizzybot.jobbot.entities;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import org.dizzybot.jobbot.enums.JobStatusEnum;
+import org.dizzybot.jobbot.enums.OutcomeStatusEnum;
+import org.springframework.util.StringUtils;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+@Entity
+@Getter
+@Setter
+public class Job {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private String title;
+
+    private String description;
+
+    @NotNull
+    private double remuneration;
+
+    @NotNull
+    private Date time; // post time?
+
+    private Date expiredDate;
+
+    private JobStatusEnum jobStatusEnum;
+
+    @OneToOne
+    @JoinColumn(name = "user_id")
```

Comment:
`name = "employer_id"`.

Reply:
so it is the name of the instance, not the class ~.~